app/views/users/two_factor_authentication_setup/index.html.slim

```diff
@@ -24,4 +24,4 @@ p.mt-tiny.mb0
   = render 'users/shared/otp_delivery_preference_selection'
   = f.button :submit, t('forms.buttons.send_security_code')
 
-= render 'shared/cancel', link: destroy_user_session_path
+= render 'shared/cancel', link: destroy_user_session_path, decorated_session: decorated_session
```

Comment:
No need to pass in the decorated_session here since it's already available to the view automatically by being defined as a `helper_method` in ApplicationController.